neon/tests/natives/qt_compat.pm

```diff
@@ -61,6 +61,10 @@ sub run {
 
     # install packages
     select_console 'log-console';
+    if ( testapi::get_var("ADD_TESTING_REPO") ) {
```

Comment:
oh, excess spacing here. and you don't need to testapi:: here, note the `use testapi` above

Reply:
fixed too..